misc/init.d/cfengine3.in

```diff
@@ -525,20 +526,33 @@ cf_stop_core_daemons()
 
 cf_status_core_daemons()
 {
+    local cf_status_core_daemons_exit=0
```

Comment:
`local` is a bashism, but I won't object since I see it used in other places in this monstrous script.